--- buildroot/package/openssl/openssl.mk	2015-08-12 08:02:39.000000000 +0200
+++ buildroot/package/openssl/openssl.mk.new	2015-08-12 08:35:34.000000000 +0200
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-OPENSSL_VERSION = 1.0.2a
+OPENSSL_VERSION = 1.0.2d
 OPENSSL_SITE = http://www.openssl.org/source
 OPENSSL_LICENSE = OpenSSL or SSLeay
 OPENSSL_LICENSE_FILES = LICENSE
